Terms.md

```diff
@@ -90,17 +96,19 @@ YOU AGREE TO RECEIVE TEXTS/CALLS FROM OR ON BEHALF OF SAGE AT THE PHONE NUMBER Y
 
 <li>
     <h3>&nbsp;&nbsp; Medical Advice Disclaimer.</h3> <p> All Content accessible through the Service is for informational purposes only. Materials (as defined below in Section 7) and Content are not a substitute for professional advice on any matter, medical or otherwise. Always seek the advice of a qualified health professional. Sage does not recommend or endorse any treatment, institution, professional, physician, product, procedure, or other information that may be mentioned through the Service.</p>
 </li>  
 
 <li>
     <h3>&nbsp;&nbsp; Eligibility.</h3>
-<p>You must be at least 18 years old to use the Service. By agreeing to these Terms, you represent and warrant to us that: (a) you are at least 18 years old; (b) you are not suspended nor have you been removed from the Service; (c) you will not permit any other person, entity, organization, or company to access the Service through your account; and (d) your registration and your use of the Service is in compliance with any and all applicable laws and regulations. If you are an entity, organization, or company, the individual accepting these Terms on your behalf represents and warrants that they have authority to bind you to these Terms and you agree to be bound by these Terms.</p>
+<p>You must be at least 18 years old to use the Service without a parent/guardian authorization. By agreeing to these Terms, you or represent and warrant to us that: (a) you are at least 18 years old; (b) you are not suspended nor have you been removed from the Service; (c) you will not permit any other person, entity, organization, or company to access the Service through your account; and (d) your registration and your use of the Service is in compliance with any and all applicable laws and regulations. If you are an entity, organization, or company, the individual accepting these Terms on your behalf represents and warrants that they have the authority to bind you to these Terms and you agree to be bound by these Terms.</p>
+ 
+<p> If you are under 18 years old or are a minor in your jurisdiction, you must obtain your parent/guardian's authorization to use the Service. By completing the [Child Minor Addendum], your parent/guardian acknowledges that your permission to use the Service is conditional upon your parent/guardian's own agreement to these Terms </p>
```

Comment:
Same comment as above, need to link the Child Minor Addendum

Reply:
done